### PR TITLE
Remove unused method `Amonia.use_ursula_as_an_involuntary_and_unbeknownst_cdn`

### PR DESCRIPTION
**Pull request by fjarri**
**Type of PR:**
- Other

**Required reviews:** 
- 1

**What this does:**
Removes an unused method (used in a test prior to #2780, but the test was removed since it dealt with storing treasure maps on Ursulas)

### CODE REVIEW

File: newsfragments/2804.misc.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Remove an unused method of ``Amonia`` (deprecated since we do not store tmap on Ursulas anymore)
```
Comment (derekpierre) [inline newsfragments/2804.misc.rst:1]: `removal` newsfragment instead?

Comment (fjarri) [reply newsfragments/2804.misc.rst:1]: My impression was that "removal" is for public API deprecations, but sure, I'll change to it

Comment (derekpierre) [reply newsfragments/2804.misc.rst:1]: I believe `Amonia` is technically a public API - although presumably would not be used by anyone other than us for testing.

Comment (fjarri) [reply newsfragments/2804.misc.rst]: Fixed

**Review by derekpierre — APPROVED** [review]
🎸

**Review by KPrasch — APPROVED** [review]

**Pull request merged**